chat.md

```diff
@@ -1,9 +1,22 @@
 ---
 layout: page
-title: "IRC"
-permalink: /irc/
-redirect_from: /catholic-irc-chat-channel/
+title: "Chat"
+permalink: /chat/
+redirect_from:
+  - /irc/
+  - /catholic-irc-chat-channel/
 ---
+
+**We're building a community of Catholic developers.** Come join us! You can find us on Slack and on IRC.
+
+## Slack
+
+The [CatholicDevs Slack Workspace](https://join.slack.com/t/catholicdevs/shared_invite/zt-dz50teya-019q4Zake3JdmsWiTxRBBg) is a place to chat about open source Catholic projects and hang out with other Catholic developers.
```

Comment:
@JohnRDOrazio Do you have any preferences about what we say here? I don't mind updating the wording :slightly_smiling_face:

Reply:
Fine with me, @HenriqueDerosa you okay with that?

Reply:
We have a steady description in the begining of channel #seeding.
Here it is. You can change it as well.

Engage with other catholic developers across the globe and grow together by knowing each other, sharing spirituality for work and collaborating in projects